packages/react-vapor/src/components/limit/tests/Limit.spec.tsx

```diff
@@ -1,20 +1,10 @@
-import {mount, ReactWrapper} from 'enzyme';
-import {shallowWithStore} from '@helpers/enzyme-redux';
+import {render, screen} from '@test-utils';
+import userEvent from '@testing-library/user-event';
 import * as React from 'react';
-import {Provider} from 'react-redux';
-import * as _ from 'underscore';
 
-import {clearState} from '../../../utils';
-import {getStoreMock} from '../../../utils/tests/TestUtils';
-import {InputConnected} from '../../input';
 import {Limit, LimitProps} from '../Limit';
 
 describe('Limit', () => {
```

Comment:
Maybe add a test for the ability to display a limit component that has no limit (only usage)?

Reply:
is it me or the default value (line 40) `limit = 100` shouldn't be there since it can happened that no limit is set on the component?